src/_provisional.scss

```diff
@@ -263,6 +271,144 @@ $provisional-alert-success-color: #278400;
 		@extend %provisional-alert-first-child-before-disable;
 	}
 
+	&.gc-chckbxrdio {
+
+		label {
+			/* (0,2,1) */
+			cursor: pointer;
+			display: block;
+			font-size: 20px;
+		}
+
+		legend {
+			/* (0,2,1) */
+			float: none;
+			font-size: 22px;
+			font-weight: 700;
+			margin-bottom: 15px;
+			margin-top: 0;
+		}
+
+		input[type="radio"],
+		input[type="checkbox"] {
+			/* (0,2,0) */
+			border: 0;
+			clip: rect(0 0 0 0);
+			height: 1px;
+			margin: -1px;
+			overflow: hidden;
+			padding: 0;
+			position: absolute;
+			width: 1px;
+
+			&[disabled] + label,
+			&.disabled + label { // TODO: Pattern justification for the class `.disabled`
+				/* (0,3,3) */
+				cursor: not-allowed;
+				opacity: .5;
+			}
+
+			+ {
+				label {
+					/* (0,3,4) */
+					display: block;
+					line-height: 2;
+					margin-left: 36px;
+				}
+
+				label::before {
+					/* (0,2,2) */
+					border: 4px solid #fff;
+					box-shadow: 0 0 0 2px #000;
+					content: "";
+					display: inline-block;
+					height: 36px;
+					left: 0;
+					position: absolute;
+					top: 0;
+					width: 36px;
+				}
+
+				label:hover::before {
+					/* (0,3,1) */
+					@include provisional-chckbxrdio-hover;
+				}
+			}
+
+			&:focus + label::before {
+				/* (0,3,2) */
+				@include provisional-chckbxrdio-focus;
+			}
+		}
+
+		input[type="radio"] {
+
+			+ {
+				label::before {
+					/* (0,2,2) */
+					border-radius: 50%;
+				}
+			}
+
+			&:checked + {
+
+				label::before {
+					/* (0,3,2) */
+					background: #444;
+				}
+			}
+		}
+
+		input[type="checkbox"] {
+
+			&:checked + label::after {
+				/* (0,3,2) */
+				border-color: #333;
+				border-style: solid;
+				border-width: 0 5px 5px 0;
+				content: "";
+				display: inline-block;
+				height: 26px;
+				left: 12px;
+				position: absolute;
+				top: 2px;
+				transform: rotate(45deg);
+				width: 13px;
+			}
+
+			&.checkbox-small {
```

Comment:
This flag will work better if it can be used with the gc-chckbxrdio style. So all the containing checkbox will be consistent instead of providing an emphasis effect for some of them.